### DISCUSSION TITLE
Setting your name

### DISCUSSION THREAD

**Discussion by RayBytes** [community/community#52798] [GitHub Education]
Hey, I wanted to verify myself however I got this message:

You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

Is there any way to get verified without having to do this? Not comfortable with putting up my full name on my github account.

**Comment by RayBytes** [comment]
Bump

**Comment by samus-aran** [comment]
Hi @RayBytes 👋 

Thank you for feedback and raising your concerns! We will pass this along to our internal team. 

We are extremely proud that the GitHub Global Campus has provided _millions_ of students with access to some of the industry’s most amazing tools and offers all free of charge to compliment their studies. As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However, sometimes we may require a little more evidence from the student applying to help us verify their academic status. This can be due to the system detecting a pattern used by bad actors or simply processing a new type of evidence. While submitting requested information is voluntary, we do thank students for providing this if possible as it speeds up their verification. 
While we do take our Community’s feedback seriously we are also mindful to ensure that bad actors impersonating students don't jeopardize our ability to provide academic benefits of value. This includes the ability to momentarily compare the evidence provided against your GitHub account information.

I can confirm that once you have submitted your application you can revert this information back as the process only captures your profile details within that instance for each new submission you make. For more information on how GitHub processes and uses personal data, please see the [GitHub Privacy Statement](https://docs.github.com/en/site-policy/privacy-policies/github-privacy-statement).

**Comment by Infinitay** [comment]
Came here to say the same thing. I guess I'm stuck with using GitLab until GitHub removes this ridiculous requirement. I'm already supply my school transcript and other identifying information such as my ID. Why do you also need my name to be explicitly displayed publicly on GitHub? Isn't it already enough to be written on the transcript/current courses sheet?
